x-pack/packetbeat/magefile.go

```diff
@@ -165,7 +165,12 @@ func TestPackages() error {
 }
 
 func SystemTest(ctx context.Context) error {
-	mg.SerialDeps(getNpcapInstaller, devtools.BuildSystemTestBinary)
+	// Buildkite (CI) images have preinstalled npcap
+	if os.Getenv("CI") == "true" {
+		mg.SerialDeps(devtools.BuildSystemTestBinary)
+	} else {
+		mg.SerialDeps(getNpcapInstaller, devtools.BuildSystemTestBinary)
+	}
```

Comment:
@oakrizan Is this related to the test enablement?

Reply:
Yes, I've merged it in a separate MR. Will be removed after rebase